src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/jpa/repository/PrisonerIepLevelRepositoryTest.kt

```diff
@@ -167,4 +167,61 @@ class PrisonerIepLevelRepositoryTest : TestBase() {
       assertThat(repository.findAllByBookingIdOrderByReviewTimeDesc(secondBookingId).toList()).hasSize(2)
     }
   }
+
+  @Test
+  fun `counts prisoners on a level`(): Unit = runBlocking {
```

Comment:
The test description can probably be adjusted to something along the lines of `"checks if any prisoners on level"` (or something like that)

Reply:
oops. fixed in #426